apps/db/lib/db_schema/statement.ex

```diff
@@ -18,6 +22,42 @@ defmodule DB.Schema.Statement do
   @required_fields ~w(text time video_id)a
   @optional_fields ~w(speaker_id)a
 
+  # Define queries
+
+  @doc """
+  Select all statements and order them by id.
+
+  ## Params
+
+    * query: an Ecto query
+    * filters: a list of tuples like {filter_name, value}.
+      Valid filters:
+        - commented: select all statements without comments if commented == false, otherwise select those with comments
+    * limit: Max number of videos to return
+  """
+  def query_list(query, filters \\ [], limit \\ nil) do
+    query
+    |> order_by([s], desc: s.id)
+    |> filter_with(filters)
+    |> limit_statement_query_list(limit)
+  end
+
+  # TODO: Not sure if it is used ...
+  defp limit_statement_query_list(query, nil),
+    do: query
+
+  defp limit_statement_query_list(query, limit),
+    do: limit(query, ^limit)
+
+  defp filter_with(query, filters) do
+    Enum.reduce(filters, query, fn
+      {:commented, false}, query ->
+        from(s in query, left_join: c in assoc(s, :comments), where: is_nil(c.statement_id))
+      {:commented, true}, query ->
+        from(s in query, left_join: c in assoc(s, :comments), where: not is_nil(c.statement_id))
```

Comment:
There's a problem with this approach: if a statement has 3 comments, it will be returned 3 times because the SQL result will be something like:

<table>
<tr>
	<td>Statement ID
	<td>Comment ID
<tr>
	<td>1
	<td>1
<tr>
	<td>1
	<td>2
<tr>
	<td>1
	<td>3
</table>

We need a `GROUP BY` or similar mechanism to solve that